runtime/vault-filesystem/src/test/java/io/aklivity/zilla/runtime/vault/filesystem/internal/FileSystemVaultTest.java

```diff
@@ -80,4 +83,21 @@ public void shouldResolveClient() throws Exception
         assertThat(signedKeys.length, equalTo(1));
         assertThat(signedKeys[0], not(nullValue()));
     }
+
+    public static Path getResourcePath(
+        String resource)
+    {
+        URL url = FileSystemVaultTest.class.getResource(resource);
+        assert url != null;
+        Path path = null;
+        try
+        {
+            path = Path.of(url.toURI());
+        }
+        catch (Exception ex)
+        {
+            rethrowUnchecked(ex);
+        }
+        return path;
```

Comment:
```suggestion
        assert url != null;
        return Path path = Paths.get(URI.create(url.toString()));
```

Does this help to simplify?

Reply:
thx, done